src/test/mockData.ts

```diff
@@ -112,7 +112,7 @@ export const mockData: MockCity =
                               }
                             },
                             startTime:[9,0],
-                            endTime:[15,0],
+                            endTime:[11,0],
                             link: "http://..."
                         }]
             },
```

Comment:
If ideas is the only object on that json, then that last coma shouldn't be there either. ❤️